src/Akka.Persistence.Sql.TagTableMigration/Options.cs

```diff
@@ -65,8 +65,8 @@ public static string ToHocon(this ProviderType type)
                 ProviderType.SqlServer2019 => ProviderName.SqlServer2019,
                 ProviderType.SqlServer2022 => ProviderName.SqlServer2022,
                 ProviderType.MySql => ProviderName.MySql,
-                ProviderType.MySqlOfficial => ProviderName.MySqlOfficial,
-                ProviderType.MySqlConnector => ProviderName.MySqlConnector,
+                ProviderType.MySqlOfficial => ProviderName.MySql80MySqlData,//.MySqlOfficial,
+                ProviderType.MySqlConnector => ProviderName.MySql80MySqlConnector,
```

Comment:
Should we add `ProviderType` changes here for the Tag Table migration Big difference is things are now a bit more explicit for MySQL versions...

Not sure if this implies config updates required (We could always try to translate if needed I guess?

Reply:
Either we add all of the new `ProviderName` constant names and remove the missing ones, or we just have the user provide the proper string value for their target provider and remove `ProviderType` altogether. 

This isn't really sensitive to breaking change as it is only used as a rather simple string argument to enum to const transform, it is not used anywhere else.